install.sh

```diff
@@ -135,23 +136,23 @@ create_tfstate_resources() {
 
 # Deploy the infrastructure if is not created yet otherwise apply the changes to existing infrastructure
 create_update_infrastructure() {
-  Echo "Starting to analyze the infrastructure..."
-  if ! test -d "${ROOT_PATH}/.terraform" ; then
-    echo "Migrating the terraform state to S3 bucket..."
+  log "Starting to analyze the infrastructure..."
+  if [ ! -d "${ROOT_PATH}/.terraform" ]; then
+    log "Running Migrating the terraform state to S3 bucket..."
     terraform -chdir="${ROOT_PATH}" init -migrate-state | tee -a "${LOG_FILE}"
     terraform -chdir="${ROOT_PATH}" init | tee -a "${LOG_FILE}"
   fi
-  terraform -chdir="${ROOT_PATH}" apply -auto-approve "${OVERRIDE_CONFIG_FILE}" | tee -a "${LOG_FILE}"
+  terraform -chdir="${ROOT_PATH}" apply -auto-approve -no-color "${OVERRIDE_CONFIG_FILE}" | tee -a "${LOG_FILE}"
```

Comment:
Thanks for this change 👍 , Also I've added `-no-color` as part of TFKUBE-192, but as you applied it here I am happy to revert my change.